lib/location/location_core.c

```diff
@@ -14,10 +15,13 @@
 #include "method_gnss.h"
 #endif
 #if defined(CONFIG_LOCATION_METHOD_CELLULAR)
-#include "method_cellular.h"
+#include "scan_cellular.h"
```

Comment:
Overall, please compile only with CONFIG_LOCATION_METHOD_CELLULAR/WIFI separately.
It doesn't compile. Not directly due to this PR but worth to be fixed.

See for example CONFIG_LOCATION_METHOD_CELLULAR_RECV_BUF_SIZE usage/Kconfig definition and https://github.com/nrfconnect/sdk-nrf/blob/main/lib/location/Kconfig#L205

Reply:
I've fixed this now with Kconfig modifications and existing receive buffer sizes are replaced by `CONFIG_LOCATION_SERVICE_CLOUD_RECV_BUF_SIZE`